recipes/autoconf/all/test_package/conanfile.py

```diff
@@ -1,53 +1,51 @@
-import shutil
-from os import path
-
 from conan import ConanFile
 from conan.tools.build import can_run
-from conan.tools.gnu import Autotools
-from conan.tools.microsoft import unix_path
-
-required_conan_version = ">=1.50.0"
+from conan.tools.env import Environment, VirtualBuildEnv
+from conan.tools.files import copy
+from conan.tools.gnu import Autotools, AutotoolsToolchain
+from conan.tools.layout import basic_layout
+from conan.tools.microsoft import is_msvc
+import os
 
 
 class TestPackageConan(ConanFile):
-    settings = "os", "compiler", "build_type", "arch"
-    exports_sources = "configure.ac", "config.h.in", "Makefile.in", "test_package_c.c", "test_package_cpp.cpp",
-    generators = "AutotoolsDeps", "AutotoolsToolchain", "VirtualBuildEnv"
+    settings = "os", "arch", "compiler", "build_type"
     test_type = "explicit"
+    win_bash = True
```

Comment:
Maybe move this to the build_requirements method for consistency

Reply:
no, it doesn't work in test package